h/static/styles/partials-v2/_search.scss

```diff
@@ -249,11 +261,10 @@
 }
 
 .search-bucket-stats__incontext-link {
-  display: block;
-  margin-top: 5px;
-
   color: $grey-6;
-  font-weight: bold;
+  &:hover {
+    text-decoration: underline;
```

Comment:
Just noting that, unless I'm missing the pattern, we seem to be quite inconsistent about whether links are underlined or not and whether they get an underline when hovered or not. I think 3/4 possible combinations of those two are present on the search page alone! Not this PR's fault though, I think it has already happened